srcpkgs/jansson/template

```diff
@@ -1,20 +1,16 @@
 # Template file for 'jansson'
 pkgname=jansson
-version=2.13.1
+version=2.14
 revision=1
 build_style=gnu-configure
 hostmakedepends="autoconf automake libtool"
```

Comment:
```suggestion
```
this is not required now, because `pre_configure()` was removed

Reply:
Thanks for letting me know. I made the suggested changes and pushed them.